workshops/erc20-mts/scripts/cli-tx.sh

```
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+source .local.env
+
+injectived tx $* --keyring-backend=test --chain-id $CHAIN_ID --node $INJ_URL --gas=auto --gas-adjustment=1.3 --gas-prices=10inj
```

Shellcheck (0.10.0)

[warning] Use "$@" (with quotes) to prevent whitespace problems.

(SC2048)